processing/data_collection/gazette/spiders/pi_teresina.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+from urllib.parse import urlencode
+
+import dateparser
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PiTeresina(BaseGazetteSpider):
+    TERRITORY_ID = "2211001"
+    BASE_URL = "http://dom.pmt.pi.gov.br/lista_diario.php"
+    name = "pi_teresina"
+    allowed_domains = ["dom.pmt.pi.gov.br"]
+    start_urls = [BASE_URL]
+
+    def parse(self, response):
+        total_items = int(response.css(".texto span b::text").get())
+        total_pages = self.calculate_number_of_pages(total_items, 10)
+
+        for i in range(1, total_pages + 1):
+            params = {"pagina": i}
+            url = "{}?{}".format(self.BASE_URL, urlencode(params))
```

Comment:
Nothing wrong here. But I think you can use f-strings. Something like this:

```suggestion
            url = f"{self.BASE_URL}?{urlencode(params)}"
```

Reply:
Done.